src/microdot/microdot.py

```diff
@@ -786,7 +789,7 @@ def send_file(cls, filename, status_code=200, content_type=None,
         if max_age is not None:
             headers['Cache-Control'] = 'max-age={}'.format(max_age)
 
-        if compressed:
+        if compressed or filename.endswith('.gz'):
```

Comment:
```suggestion
        if compressed:
```

I think this is fine as it was. If the user passes `compressed=False` then the response should not be compressed.